frontend/common/src/components/Button/Button.tsx

```diff
@@ -195,7 +195,7 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         }}
         {...rest}
       >
-        {children}
+        Download
```

Comment:
Is there a reason why you have made this a static value for a reusable component like Button? I am confused.

Reply:
Might this PR be related to https://github.com/GCTC-NTGC/gc-digital-talent/issues/3409? If so, there is a PR that has resolved that issue: https://github.com/GCTC-NTGC/gc-digital-talent/pull/3421.